packages/serverless/test/util/AbstractServerlessControl.js

```diff
@@ -16,10 +16,6 @@ const retry = require('@instana/core/test/test_util/retry');
 // const delay = require('@instana/core/test/test_util/delay');
 const config = require('@instana/core/test/config');
 
-// To address the certificate authorization issue with node-fetch, process.env.NODE_TLS_REJECT_UNAUTHORIZED
-// was set to '0'. Refer to the problem discussed in https://github.com/node-fetch/node-fetch/issues/19
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
If we have to set this env, then we set it in the mocha test config file. Not here.